### DISCUSSION TITLE
Modifying other users' Weekly Hours

### DISCUSSION THREAD

**Discussion by spenol** [kimai/kimai#5044] [General] [answered]
Hi,

I'd like to be able to easily update other users' weekly hours rather than having to individually create entries per day in the 'All times' screen. 

For example, we have an employee that has gone on leave and we want to just assign 8 hours per day for 2 weeks which is really easy with the 'Weekly hours' screen for the user that's logged in, but for other users you have to manually create entries for each day in the 'All times' screen which is really time consuming.

Is there an easy way to do this? Ideally it would be great to select another user on the Weekly hours screen, like you can do in the 'Employment contract -> working hours' screen.

Am I missing an easy way to do this?

Thanks,

Oliver.

**Comment by kevinpapst** [comment, marked as answer]
Test the demo. Update your Kimai. Be happy.

**Reply by spenol** [reply]
Thank you @kevinpapst
